build-system/tasks/make-extension/template/bento/extensions/amp-__component_name_hyphenated__/__component_version__/amp-__component_name_hyphenated__.js

```diff
@@ -11,7 +11,9 @@ class Amp__component_name_pascalcase__ extends BaseElement {
   /** @override */
   init() {
     // __do_not_submit__: This is example code only.
-    this.registerApiAction('exampleToggle', (api) => api./*OK*/exampleToggle());
```

Comment:
note: if you wanted to, you could keep this helper but put it in Core instead of living in PBE

Reply:
I decided against that since the syntax would not really be smaller. Passing `this` is kind of weird also.

```diff
- this.registerAction('exampleToggle', () => this.api()./*OK*/exampleToggle());
+ registerApiAction(this, 'exampleToggle', (api) => api./*OK*/exampleToggle());